_build.sh

```diff
@@ -6,5 +6,5 @@ printf " ! USE ./mach FOR MORE OPTIONS                 !\n\n"
 
 mkdir build
 cp config/default_config.cmake build/config.cmake
-cmake -B build -G"Ninja Multi-Config"
+cmake -B build -G"Ninja Multi-Config" -DBUILD_LIBCRUX=ON
```

Comment:
Is this used anywhere where we want libcrux on?
I don't think we want libcrux here just yet.